hack/ci-kind-molecule-tests.sh

```diff
@@ -114,7 +114,7 @@ Options:
 
 -sd|--src-dir <directory>
     Where the git source repositories will be cloned.
-    Default: /tmp/KIALI-GIT
+    Default: /tmp/KIALI-GIT-KIND
```

Comment:
Same with this?

Reply:
no. branch name and file name are irrelevant to each other

Reply:
Oh, I see what you are asking... because this is only running with KinD, I put this is a KIND specific directory so as not to conflict with the other CI hack script that uses KIALI-GIT -- I don't want the test logs stepping on each other so I set this default name to be different.